docs/source-pytorch/conf.py

```diff
@@ -146,6 +146,8 @@ def _transform_changelog(path_in: str, path_out: str) -> None:
 # myst-parser, forcing to parse all html pages with mathjax
 # https://github.com/executablebooks/MyST-Parser/issues/394
 myst_update_mathjax = False
+# https://myst-parser.readthedocs.io/en/latest/syntax/optional.html?highlight=anchor#auto-generated-header-anchors
+myst_heading_anchors = 3
```

Comment:
This fixes the failure in CI. I added it to the apps conf too just in case, even though it doesn't link to heading anchors for now.